tasks/db_deploy/install/orca_sql.py

```diff
@@ -140,10 +137,10 @@ def orca_schema_sql() -> TextClause:
         COMMENT ON SCHEMA orca
             IS 'Contains all the objects needed to operate the ORCA application';
 
-        -- GRANT the privelages needed
+        -- GRANT the privileges needed
         GRANT USAGE ON SCHEMA orca TO orca_app;
 
-        -- Setup Default Privelages for application user as a catch all
+        -- Setup Default Privileges for application user as a catch all
```

Comment:
privileges

Reply:
Yup. That's what I changed to.

Reply:
That's what I changed to. Unless you're suggesting that it should be lower-case?

Reply:
I think I saw lowercase privileges somewhere. Nevermind,